statics/js/components/graph-layout.js

```diff
@@ -75,6 +253,22 @@ var TopologyGraphLayout = function(vm, selector) {
 
 TopologyGraphLayout.prototype = {
 
+  linkLabelFactory: function(link) {
```

Comment:
can you rename the "updateBandwidth" function to something else as it handles two things now latency + bandwidth

Reply:
changing to `updateLinkLabel`